vision/beta/cloud-client/src/main/java/com/example/vision/Detect.java

```diff
@@ -1423,4 +1438,239 @@ public static void detectDocumentsGcs(String gcsSourcePath, String gcsDestinatio
+
+      // Display the results
+      for (AnnotateImageResponse res : responses) {
+        for (LocalizedObjectAnnotation entity : res.getLocalizedObjectAnnotationsList()) {
+          out.format("Object name: %s\n", entity.getName());
+          out.format("Confidence: %s\n", entity.getScore());
+          out.format("Normalized Vertices:\n");
+          entity
+              .getBoundingPoly()
+              .getNormalizedVerticesList()
+              .forEach(vertex -> out.format("- (%s, %s)\n", vertex.getX(), vertex.getY()));
+        }
+      }
+    }
+  }
+  // [END vision_localize_objects]
+
+  // [START vision_localize_objects_uri]
+  /**
+   * Detects localized objects in a remote image on Google Cloud Storage.
+   *
+   * @param gcsPath The path to the remote file on Google Cloud Storage to detect localized objects
+   *     on.
+   * @param out A {@link PrintStream} to write detected objects to.
+   * @throws Exception on errors while closing the client.
+   * @throws IOException on Input/Output errors.
+   */
+  public static void detectLocalizedObjectsGcs(String gcsPath, PrintStream out)
+      throws Exception, IOException {
+    List<AnnotateImageRequest> requests = new ArrayList<>();
+
+    ImageSource imgSource = ImageSource.newBuilder().setGcsImageUri(gcsPath).build();
+    Image img = Image.newBuilder().setSource(imgSource).build();
+
+    AnnotateImageRequest request =
+        AnnotateImageRequest.newBuilder()
+            .addFeatures(Feature.newBuilder().setType(Type.OBJECT_LOCALIZATION))
+            .setImage(img)
+            .build();
+    requests.add(request);
+    // Perform the request
+    try (ImageAnnotatorClient client = ImageAnnotatorClient.create()) {
+      BatchAnnotateImagesResponse response = client.batchAnnotateImages(requests);
+      List<AnnotateImageResponse> responses = response.getResponsesList();
+      client.close();
+      // Display the results
+      for (AnnotateImageResponse res : responses) {
+        for (LocalizedObjectAnnotation entity : res.getLocalizedObjectAnnotationsList()) {
+          out.format("Object name: %s\n", entity.getName());
+          out.format("Confidence: %s\n", entity.getScore());
+          out.format("Normalized Vertices:\n");
+          entity
+              .getBoundingPoly()
+              .getNormalizedVerticesList()
+              .forEach(vertex -> out.format("- (%s, %s)\n", vertex.getX(), vertex.getY()));
+        }
+      }
+    }
+  }
+  // [END vision_localize_objects_uri]
+
+  // [START vision_handwritten_ocr]
+  /**
+   * Performs handwritten text detection on a local image file.
+   *
+   * @param filePath The path to the local file to detect handwritten text on.
+   * @param out A {@link PrintStream} to write the results to.
+   * @throws Exception on errors while closing the client.
+   * @throws IOException on Input/Output errors.
+   */
+  public static void detectHandwrittenOcr(String filePath, PrintStream out) throws Exception {
```

Comment:
I've updated the sample image: https://storage.googleapis.com/cloud-samples-data/vision/handwritten.jpg